index.html

```diff
@@ -18,18 +31,23 @@
                 </div>
 
                 <nav class="desktop-nav">
+                    <div class="x-nav"><img src="./images/Xmark (2).png" alt=""></div>
                     <ul>
-                        <li><a href="#">Portfolio</a></li>
-                        <li><a href="#">About</a></li>
-                        <li><a href="#">Contact</a></li>
+                        <li><a href="#ancre1">Portfolio</a></li>
+                        <li><a href="#ancre2">About</a></li>
+                        <li><a href="#ancre3">Contact</a></li>
                     </ul>
```

Comment:
- [ ] Based on one of the requirements, the hamburger button and menu should follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, tests, margins). Therefore, please make sure that your design is the same as provided in Figma:

Your design  | Figma design
------------- | -------------
<img width="431" alt="Screenshot 2022-03-07 at 15 51 38" src="https://user-images.githubusercontent.com/77038610/157029293-9b9e9daf-0b22-4d99-b8a7-75b90ff0ff88.png"> | <img width="250" alt="Screenshot 2022-03-07 at 15 52 00" src="https://user-images.githubusercontent.com/77038610/157029357-e9bf3ace-8056-4ce2-9783-b5193dc3f954.png">

You can start with setting background color to `#6070FF` and setting the font size to `32px`😉

Reply:
Thank you for you feedback, 

I really appreciate your work and you will see I took time to respect each point, but for the background, when I set it to the one provided by figma design it removes the backdrop-filter. That's actually why it's still different.
Hope you understand @ShoiraTa 

Happy coding !!

Reply:
@AnicetFantomas, thank you for telling about that, in this case, you can try to select some color that is close to the provided one

Reply:
issue well resolved 👍